metrics/configs/presubmit-health.yaml

```diff
@@ -0,0 +1,91 @@
+metric: presubmit-health
+description: Calculate daily health metrics for presubmits
+query: |
+  #standardSQL
+  SELECT
+    UNIX_SECONDS(day) day,
+    job,
+    ROUND(prs_failed/prs * 100, 1) pr_failure_perc,
+    prs,
+    prs_failed,
+    total_runs total_runs,
+    ROUND(total_elapsed / total_runs / 60, 1) avg_run_time_minutes,
```

Comment:
Consider using seconds granularity and converting to minutes in grafana. That will make the daily results that are dumped to GCS more accurate for potential consumers.  It will also be better for shorter lived jobs (which we don't really have, but others might).

Reply:
Extra precision in averages isn't necessarily useful-- even for short jobs, how much do you care about +/- 6 seconds?

Reply:
Well its +/- 30 seconds since we are rounding to the minute.  I don't think we care about that, but someone running lightweight jobs that complete quickly might. Is there some benefit to throwing away the precision?

This doesn't matter much though. Feel free to ignore if you don't want to change it.

Reply:
we're rounding to 1 decimal place = .1 minute accuracy = +/- 6 seconds :)

the primary benefit is the JSON files are more human readable

Reply:
Ah, thats much more reasonable then.
.1 minute accuracy is actually +/- 3 seconds (a range of 6 seconds) so thats even better.
Thanks for clarifying.